ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/rules/views/HiveAggregatePartitionIncrementalRewritingRule.java

```diff
@@ -0,0 +1,151 @@
+package org.apache.hadoop.hive.ql.optimizer.calcite.rules.views;/*
```

Comment:
how did this header pass on the precommit?
but any way; could you fix this?

Reply:
Fixed